.changeset/gentle-melons-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@skeletonlabs/skeleton": patch
+---
+
+Replaced manual subscription with automatic subscriptions handled by svelte
```

Comment:
Need to prefix your summary with `bugfix:`. It follows the same convention as the PR name itself.

Reply:
Looks like you went offline for the day, so I'll take care of it this time!